src/main/resources/META-INF/lsp4ij-quarkus.xml

```diff
@@ -1,20 +1,14 @@
 <idea-plugin>
   <extensions defaultExtensionNs="com.redhat.devtools.lsp4ij">
+
     <!-- Quarkus LSP -->
     <server id="microprofile"
-            label="Tools for MicroProfile"
+            bundle="messages.MicroProfileBundle"
+            labelKey="microprofile.server.label"
```

Comment:
java.lang.Throwable: 'microprofile.server.label' is not found in java.util.PropertyResourceBundle@1282d454(messages.MicroProfileBundle)